SimG4CMS/Muon/src/MuonSensitiveDetector.cc

```diff
@@ -20,72 +19,71 @@
 #include "SimG4Core/Notification/interface/G4TrackToParticleID.h"
 #include "SimG4Core/Physics/interface/G4ProcessTypeEnumerator.h"
 
-#include "G4SDManager.hh"
 #include "G4VProcess.hh"
 #include "G4EventManager.hh"
 #include "G4SystemOfUnits.hh"
+#include "G4Step.hh"
+#include "G4StepPoint.hh"
+#include "G4Track.hh"
 
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
 #include <iostream>
 
+//#define DebugLog
```

Comment:
@civanch I understand that you keep the revision of the verbosity that we discussed for a further PR, am I correct?

Reply:
In this code it turned out that a significant part of computation is done only for debugging. It much less work is to include #ifdef, LogDebug itself cannot serving for that.

Reply:
@fabiocos , in this PR I tried improving verbosity using Verbatim stream and LogDebug where possible but in two places seems the best solution is to add #ifdef

Reply:
@civanch I was referring to the standardisation to EDM_ML_DEBUG...